packages/core/src/components/BaseInput/__stories__/BaseInput.stories.tsx

```diff
@@ -3,15 +3,17 @@ import { createComponentTemplate } from "vibe-storybook-components";
 import BaseInput from "../BaseInput";
 
 const metaSettings = createStoryMetaSettingsDecorator({
-  component: BaseInput
+  component: BaseInput,
+  isInternal: true,
+  shouldCreateAutodocsPage: true
```

Comment:
just to make the magic "backstage" instead of declaring `tags: ["internal", "autodocs"]`.

Reply:
I wish we could just add `internal: true` and that's it, and get the autodocs page, we need to "register" it in 3 different properties

Reply:
@talkor done
autodocs is now opt-out optional if internal is passed